src/plugins/crypto/README.md

```diff
@@ -0,0 +1,57 @@
+- infos = Information about crypto plugin is in keys below
+- infos/author = Peter Nirschl <peter.nirschl@gmail.com>
+- infos/licence = BSD
+- infos/needs = 
+- infos/provides = storage
```

Comment:
This information is wrong. (The testcases also fail because of that)

There is no real category for such a plugin up to now. Maybe filefilter?